share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -136,10 +136,12 @@ A Symbol (e.g. `:x11`) specifies a [`Requirement`](http://www.rubydoc.info/githu
 
 A Hash (e.g. `=>`) specifies a formula dependency with some additional information. Given a single string key, the value can take several forms:
 
-*   a Symbol (currently one of `:build`, `:optional`, `:recommended`).
+*   a Symbol (currently one of `:build`, `:devel`, `:optional`, `:recommended`).
     - `:build` means that dependency is a build-time only dependency so it can
       be skipped when installing from a bottle or when listing missing
       dependencies using `brew missing`.
+    - `:devel` generates an implicit `devel` option, meaning that the development
+      version of the dependency will be installed.
```

Comment:
I don't see the use case for this. It smells like feature patching. If the software require's a devel dependency, it should be in an explicit development spec of the main software.